content/vendor/additional-objects/config.md

```diff
@@ -0,0 +1,8 @@
+---
+date: 2019-11-27
+linktitle: "Config"
+title: Config
+weight: 20020
+---
+
+Config object specifications control the presentation of a [custom configuration page](/vendor/config/config-screen/), enabling operators to specify any required or optional application-specific config settings at install time, or update those options later through the kotsadm dashboard.
```

Comment:
I’d prefer to stick to using the term Admin Console (capitalized) over “kotsadm”. So this sentence should end with “Admin Console Dashboard” instead of “kotsadm dashboard”